packages/@okta/i18n/src/properties/login.properties

```diff
@@ -513,10 +513,11 @@ enroll.totp.enterCode = Enter code displayed from the application
 
 ## Barcode View
 # {0} is the name of a MFA factor
-enroll.totp.setupApp = Launch {0} application on your mobile device and select Add an account.
+enroll.totp.setupApp = Launch {0} on your mobile device and select “Add an account”. Scan the QR code to continue.
```

Comment:
can some test be added to verify this text?

Reply:
added.